.github/workflows/notebooks_dispatch.yml

```diff
@@ -43,6 +43,7 @@ jobs:
       - name: Install dependencies
         run: |
           poetry install --with test --with dev --all-extras
+          pip install git+https://github.com/AstarVienna/ScopeSim_Data.git
```

Comment:
```suggestion
          poetry add git+https://github.com/AstarVienna/ScopeSim_Data.git
```

Reply:
I tried using `poetry add`, but decided against it because that would make changes to `poetry.lock` and `pyproject.toml`. The goal is to simply install ScopeSim_Data, not to add it as a dependency to ScopeSim.

Using poetry or pip shouldn't matter w.r.t. running the tests, but I prefer to keep the git repository clean, unless the changes are intentional (e.g. for an assist PR).

Maybe it would be worthwhile to add ScopeSim_Data as a true dependency of ScopeSim at some point, but that is a different discussion, and would require ScopeSim_Data to be better versioned (maybe even a PyPI project).

Reply:
The diff:
```diff
$ poetry add git+https://github.com/AstarVienna/ScopeSim_Data.git
[...]
$ git diff
diff --git a/poetry.lock b/poetry.lock
index 8af0f52..b50a85b 100644
--- a/poetry.lock
+++ b/poetry.lock
@@ -3039,6 +3039,21 @@ dev = ["click", "doit (>=0.36.0)", "flake8", "mypy", "pycodestyle", "pydevtool",
 doc = ["matplotlib (>2)", "numpydoc", "pydata-sphinx-theme (==0.9.0)", "sphinx (!=4.1.0)", "sphinx-design (>=0.2.0)"]
 test = ["asv", "gmpy2", "mpmath", "pooch", "pytest", "pytest-cov", "pytest-timeout", "pytest-xdist", "scikit-umfpack", "threadpoolctl"]
 
+[[package]]
+name = "ScopeSim_Data"
+version = "0.0.1a0"
+description = "Data often used by ScopeSim and related tools."
+optional = false
+python-versions = ">=3.8"
+files = []
+develop = false
+
+[package.source]
+type = "git"
+url = "https://github.com/AstarVienna/ScopeSim_Data.git"
+reference = "HEAD"
+resolved_reference = "5a14166f91f06bdef9c7ead6ac7a7fde70dac5a7"
+
 [[package]]
 name = "scopesim-templates"
 version = "0.5.0"
@@ -3685,4 +3700,4 @@ testing = ["big-O", "jaraco.functools", "jaraco.itertools", "more-itertools", "p
 [metadata]
 lock-version = "2.0"
 python-versions = ">=3.8,<3.12"
-content-hash = "3ab0ec509f4c8c120775ca502ce187e3e9c288bff31304c0fee8e7f139e2d325"
+content-hash = "00b0114f24f38f04876b49c3ac0b8924ea12ea2975d1ca0610905916bbeff4e3"
diff --git a/pyproject.toml b/pyproject.toml
index 7840ebb..71f54e6 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -37,6 +37,7 @@ tqdm = "^4.66.1"
 synphot = "^1.1.0"
 skycalc_ipy = "^0.2.0"
 anisocado = "^0.3.0"
+scopesim-data = {git = "https://github.com/AstarVienna/ScopeSim_Data.git"}
 
 [tool.poetry.group.dev]
 optional = true
```

Reply:
I'm aware that `poetry add` modifies the lock file, my thought was that this wouldn't matter much if we don't commit that change. But I agree it would be cleaner if this could be avoided...